.devcontainer/features/languages/swift/install.sh

```
@@ -0,0 +1,198 @@
+        ;;
+    aarch64|arm64)
+        SWIFT_ARCH="aarch64"
+        ;;
+    *)
+        echo -e "${RED}Unsupported architecture: $ARCH${NC}"
+        exit 1
+        ;;
+esac
+
+# Detect Ubuntu version
+UBUNTU_VERSION=$(. /etc/os-release && echo "$VERSION_ID" | sed 's/\.//')
```

Shellcheck (0.11.0)

[warning] UBUNTU_VERSION appears unused. Verify use (or export if used externally).

(SC2034)